csrc/executor.cpp

```diff
@@ -1247,11 +1247,22 @@ LaunchParams FusionExecutor::computeLaunchParams(
     const int welford_factor =
         kernel_summary.has_block_welford || kernel_summary.has_grid_welford ? 3
                                                                             : 1;
+    // in outer reduction, may group iteration domain, e.g. when vectorized.
+    const int grouped_iter_factor = kernel_summary.has_iter_grouped_reductions
```

Comment:
wasn't the default just 1? don't see why we needed to branch here.

Meanwhile, is it safe to assert `kernel_summary.has_iter_grouped_reductions || grouped_iter_factor == 1`?

Reply:
Good point! I think I was following the welford factor, but we don't need to.
Corrected.
assert is changed to 
```
    NVF_CHECK(
        !(kernel_summary.has_iter_grouped_reductions && welford_factor == 3),
        "can't have welford and iter grouped reductions at the same time! Should be handled by grouped welford!");

```